modules/nf-core/kaiju/mkfmi/meta.yml

```
@@ -0,0 +1,42 @@
+name: "kaiju_mkfmi"
+description: Make Kaiju FMI-index file from a protein FASTA file
+keywords:
+  - classify
+  - metagenomics
+  - fastq
+  - taxonomic profiling
+tools:
+  - "kaiju":
+      description: "Fast and sensitive taxonomic classification for metagenomics"
+      homepage: "https://bioinformatics-centre.github.io/kaiju/"
+      documentation: "https://github.com/bioinformatics-centre/kaiju/blob/master/README.md"
+      tool_dev_url: "https://github.com/bioinformatics-centre/kaiju"
+      doi: "10.1038/ncomms11257"
+      licence: ["GNU GPL v3"]
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'test', single_end:false ]`
+  - fasta:
+      type: file
+      description: Protein FASTA file
```

```suggestion
      description: Uncompressed Protein FASTA file (mandatory)
```

Possibly optional, given my comment above

Adding the mandatory is per https://nf-co.re/docs/contributing/modules#documentation point 12